dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/security/config/DhisWebApiWebSecurityConfig.java

```diff
@@ -502,7 +502,8 @@ private void configureMatchers( HttpSecurity http )
                     .sessionFixation().migrateSession()
                     .sessionCreationPolicy( SessionCreationPolicy.ALWAYS )
                     .enableSessionUrlRewriting( false )
-                    .maximumSessions( 10 )
+                    .maximumSessions(
+                        Integer.parseInt( dhisConfig.getProperty( ConfigurationKey.MAX_SESSIONS_PER_USER ) ) )
```

Comment:
Would be nice with a `DhisConfigurationProvider#getInteger` method to hide the cast.